src/test/unit/base/services/authenticationService.test.ts

```diff
@@ -79,6 +94,7 @@ suite('AuthenticationService', () => {
         resetTransientErrors: sinon.fake(),
         connectionRetryLimitExhausted: false,
       } as ISnykCodeErrorHandler,
+      languageClient as unknown as LanguageClient,
```

Comment:
I don't think you need `as unknown as LanguageClient` cast here and below, as it's casted already in setup function and assigned to the type.